CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -2018,7 +2018,14 @@ protected virtual bool CreateOffscreenBrowser(Size size)
         protected virtual IWindowInfo CreateOffscreenBrowserWindowInfo(IntPtr handle)
         {
             var windowInfo = new WindowInfo();
-            windowInfo.SetAsWindowless(handle);
+            if (handle == IntPtr.Zero)
+            {
+                windowInfo.SetAsWindowless(handle);
+            }
+            else
+            {
+                windowInfo.SetAsChild(handle);
```

Comment:
The `handle` in this case is the `HWND` for the `WPF Window`, you've effectively overridden the contents of the `Window`. There are many other changes that are required like resizing/minamize/maxamise hence the creation of https://github.com/cefsharp/CefSharp.Wpf.HwndHost